e2e/app/feeoraclev2.go

```diff
@@ -0,0 +1,45 @@
+package app
+
+import (
+	"context"
+
+	"github.com/omni-network/omni/lib/contracts/feeoraclev2"
+	"github.com/omni-network/omni/lib/errors"
+	"github.com/omni-network/omni/lib/log"
+)
+
+func DeployFeeOracleV2(ctx context.Context, def Definition) error {
+	if err := deployFeeOracleV2(ctx, def); err != nil {
+		return errors.Wrap(err, "deploy fee oracle v2")
+	}
+
+	return nil
+}
+
+func deployFeeOracleV2(ctx context.Context, def Definition) error {
+	_, ok := def.Testnet.OmniEVMChain()
+	if !ok {
+		return errors.New("no omni evm chain")
+	}
+
+	allChains := def.Testnet.EVMChains()
+	for _, chain := range allChains {
+		backends := def.Backends()
```

Comment:
```suggestion
	allChains := def.Testnet.EVMChains()
	backends := def.Backends()
	for _, chain := range allChains {
```
backends initialized on every loop iteration, move outside loop.

Reply:
Moved outside of loop